sdk-incubator-lib/src/main/java/cash/z/ecc/android/sdk/model/ZatoshiExt.kt

```diff
@@ -95,5 +87,11 @@ fun BigDecimal.convertFiatDecimalToFiatString(
                 }
             }
         }
-    }.format(this)
+    }.format(this).let {
+        if (includeSymbols) {
+            it
+        } else {
+            it.replace(fiatCurrency.symbol, "")
```

Comment:
This won't remove spaces from the output. For example, the currency format for the CANADA_FRENCH locale is something like "1,23\u00A0$", and so the "\u00A0" (non-breaking space) won't be removed.